VideoLocker/build.gradle

```diff
@@ -122,9 +122,9 @@ dependencies {
      testCompile ("org.mockito:mockito-core:1.9.5"){
          exclude group: 'org.hamcrest'
       }
-     testCompile "org.robolectric:robolectric:3.0-rc3"
-     testCompile "org.robolectric:shadows-support-v4:3.0-rc3"
-     testCompile "org.robolectric:shadows-multidex:3.0-rc3"
+     testCompile "org.robolectric:robolectric:3.0"
```

Comment:
"3.1-SNAPSHOT" doesn't seem to be working right now (repository is missing some files or something)